### ISSUE TITLE
var-spacing: do not check when it's tagged !unsafe

### ISSUE THREAD

**Issue by klando** [ansible/ansible-lint#1702]
##### Summary

When defining something like: ```tag: !unsafe "{{.Name}} [{{.ImageName}}]"```

ansible-lint complains with:  ```[206] Variables should have spaces before and after: {{ var_name }}```

However, the string probably should not be parsed and tested when there is a `!unsafe`.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible 2.9.16
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/cedric/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.3 (default, Jan 22 2021, 20:04:44) [GCC 8.3.0]

ansible-lint --version
ansible-lint 5.0.12 using ansible 2.11.1

(I've tested the latest available in quay.io/ansible/toolset) 
```

- ansible installation method: OS package
- ansible-lint installation method: OS package + quay.io/ansible/toolset docker test

##### STEPS TO REPRODUCE

```console
$ cat test.yml 
- set_fact:
        a: "{{ ok }}"
        b: !unsafe "{{oktoo}}"

$ ansible-lint test.yml 
[206] Variables should have spaces before and after: {{ var_name }}
test.yml:3
        b: !unsafe "{{oktoo}}"
```

##### Desired Behaviour

Should not produce error.

##### Actual Behaviour

```paste below
$ ansible-lint test.yml 
[206] Variables should have spaces before and after: {{ var_name }}
test.yml:3
        b: !unsafe "{{oktoo}}"
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
